### PR TITLE
Add project files

### PR DESCRIPTION
**Pull request by emy-mhmd**

### CODE REVIEW

**Comment by emy-mhmd** [comment]
task done

**Comment by RamyGendy** [comment]
the SMOTE on the whole dataframe you made has caused data leakage to your model. i.e. you model is overfit. you should do SMOTE after splitting the data. so EACH of the training set and the test set remain separate. without leakage. I recommend that you use sklearn pipeline or perform smote on each split (xtrain, xtest) with fit-transform, then feed the output to your chosen model.

**Comment by RamyGendy** [comment]
the rest of your work in overall is very good
